packages/framework-integration-tests/integration/providers/aws/setup.ts

```diff
@@ -2,10 +2,14 @@ import util = require('util')
 const exec = util.promisify(require('child_process').exec)
 import { deploy, nuke } from '../../../src/deploy'
 import { config } from 'aws-sdk'
+import { sleep } from '../helpers'
 
 before(async () => {
   await setEnv()
   await checkConfigAnd(deploy)
+  console.log('Waiting 30 seconds after deployment to let the stack finish its initialization...')
```

Comment:
Did anything change? It worked without waiting so much before... 🤔

Reply:
Yes, the WebSockets. We are adding more and more subscriptions tests and they use sockets, whose AWS resource is the main source of inconsistencies right after the stack is deployed. 

We might need to revisit this in the future to be really sure.